packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -689,6 +683,15 @@ export class AppMainContainer extends Component<
     });
   }
 
+  openDashboard(widget: VariableDefinition): void {
+    const { connection } = this.props;
+    this.emitLayoutEvent('dashboardOpen', {
```

Comment:
Is this event getting used anywhere? I think this method unused and can be removed